modules/oas-page-builder/src/services/pageBuilder.ts

```diff
@@ -186,21 +223,39 @@ export const buildOpenAPIPages = async (
           repoPath,
           apiVersion,
           resourceVersion,
+          siteUrl,
+          resourceVersions,
         });
+
         if (!isSuccessfulBuild) totalSuccess = false;
       }
+
+      // provide no resource version in this context for the base version
+      await getOASpec({
```

Comment:
I might be making a mistake at the end of the day, but I believe we don't need to have this called twice here. I think the way it was before handled this! Because no matter what, we will want to call getOASpec once for the "base" API Version spec.

Reply:
I added this second call to handle the "base" case, where there exists a resource version array, but no resource version. In this case, we will pull the latest resource version. 

In the else statement, we do not provide the `resourceVersions` array. In this case, we do not provide the version options to the redoc cli.

With that said, I think I can refactor this code in such away where I don't have 3 API calls. I'll push that change up shortly!!